### ISSUE TITLE
Week beginning Tuesday 29th of August

### ISSUE THREAD

**Issue by ZooeyMiller** [dwyl/apprenticeship#19]
So at the moment Finn and I are working on learning Elixir and Phoenix still. We spent a good chunk of last week working on this phoenix todo app: https://github.com/ZooeyMiller/phoenix-elixir-todos. It still needs work based on the user stories that we defined. It would be nice to continue playing with this but I'd also really like to feel more like I'm contributing to dwyl stuff when I'm on dwyl time. 

As a result I'm trying to look through the `learn-elixir` and `learn-phoenix-framework` repos, and their issues to see what I can help with. I've made a PR to `learn-phoenix-framework` but will carry on looking for things I can improve. We've also raised a couple of issues of improvements that could possibly be made on those repos, but would want to wait for a response/clarification of the improvements being wanted before working on them. 

Who should I be assigning to PRs on these repos?

Also, I have push access on `learn-elixir` but not on `learn-phoenix-framework` which makes it harder to work on the repo as it means I have to fork it, can't add labels/assign people etc.

I'm going to be working remotely from the Isle of Wight from Wednesday until Friday because a family member is moving out of the country and I need to say goodbye. 

We're still waiting on confirmation of rates re #17, we're also very keen to get some more direction when possible.

**Comment by iteles** [comment]
No longer relevant.

For the record, @finnhodgkin and @ZooeyMiller did an excellent job working across a number of things:
+ https://github.com/dwyl/dwyl-site
+ https://github.com/dwyl/learn-seo
+ https://github.com/dwyl/learn-google-optimize
+ https://github.com/dwyl/elixir-github-auth
+ https://github.com/dwyl/library

**Comment by nelsonic** [comment]
No. they didn't. they worked on things that we DON'T NEED!
Never confuse "busy" with _useful_, Inês.
